imi/src/main/java/org/motechproject/nms/imi/service/impl/CdrFileServiceImpl.java

```diff
@@ -744,4 +753,65 @@ public void verifyDetailFileChecksumAndCount(CdrFileNotificationRequest request)
         }
         return errors;
     }
+
+    @Override
+    @MotechListener(subjects = { CDR_CSR_CLEANUP_SUBJECT })
+    public void cleanOldCallRecords() {
+        LOGGER.info("cleanOldCallRecords() called");
+        int cdrDuration = MIN_CALL_DATA_RETENTION_DURATION_IN_DAYS;
+
+        try {
+            int durationFromConfig = Integer.parseInt(settingsFacade.getProperty(CDR_CSR_RETENTION_DURATION));
+            if (durationFromConfig < cdrDuration) {
+                LOGGER.debug(String.format("Discarding retention property from config since it is less than MIN: %d", cdrDuration));
```

Comment:
sure about retention policy? I don't think there's anything spec'd so far. I figured that 5 days is a good min to deal with debugging fires?